### PR TITLE
chore(linux): Add some unit tests for keyman_config

### PR DESCRIPTION
**Pull request by ermshiperete**
Also fix the indentation in `install_kmp.py`

### CODE REVIEW

File: linux/keyman-config/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+PYTHONPATH=.:$PYTHONPATH
+
+if [ -n "$TEAMCITY_VERSION" ]; then
+    if ! pip3 list --format=columns | grep -q teamcity-messages; then
+        pip3 install teamcity-messages
+    fi
+    python3 -m teamcity.unittestpy discover -s tests -p test_*.py
+else
+    python3 -m unittest discover -v -s tests -p test_*.py
+fi
```
Comment (darcywong00) [inline linux/keyman-config/run-tests.sh:11]: Thanks for adding unit tests!

Can you update https://github.com/keymanapp/keyman/tree/master/linux#testing

to help other developers get up and running?

Comment (darcywong00) [reply linux/keyman-config/run-tests.sh:11]: For developers, is this a matter of adding `unittest` to
https://github.com/keymanapp/keyman/tree/master/linux#linux-requirementssetup
?

File: linux/keyman-config/tests/test_install_kmp.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    def test_InstallKeyboardsToIbus_SingleKbMultipleLanguages(self):
+        # Setup
+        bus = self.mockGetIbusBus.return_value
+        keyboards = [{'id': 'foo1', 'languages': [{'id': 'en'}, {'id': 'fr'}]}]
+        # Execute
+        install_keyboards_to_ibus(keyboards, 'fooDir')
+        # Verify
+        self.mockInstallToIbus.assert_called_once_with(
+            ANY, 'en:fooDir/foo1.kmx')
```
Comment (darcywong00) [inline linux/keyman-config/tests/test_install_kmp.py:69]: Does `'fr:fooDir/foo1.kmx'` also get installed?

File: linux/keyman-config/run-tests.sh (hunk 1 above)
Comment (ermshiperete) [reply linux/keyman-config/run-tests.sh:11]: > For developers, is this a matter of adding `unittest` to
> https://github.com/keymanapp/keyman/tree/master/linux#linux-requirementssetup
> ?
This is part of the default Python package.

Done.

File: linux/keyman-config/tests/test_install_kmp.py (hunk 2 above)
Comment (ermshiperete) [reply linux/keyman-config/tests/test_install_kmp.py:69]: No. I enhanced the test.

**Pull request merged**